src/client/config.rs

```diff
@@ -100,9 +172,13 @@ impl PlatzClientConfig {
 
     /// Returns Authorization header content, possibly refreshing the current
     /// credentials.
-    pub async fn get_authorization(&self) -> Result<String, PlatzClientError> {
+    pub async fn get_authorization(&self) -> Result<(String, String), PlatzClientError> {
```

Comment:
This works great, but if you want to be more strict you can use `reqwest`'s `HeaderName` and `HeaderValue` to ensure those values don't get mixed by callers.